### ISSUE TITLE
asking update about flavor='guess'

### ISSUE THREAD

**Issue by abhilashabhardwaj** [camelot-dev/camelot#151]
Hello Vinayak,

Thanks for your work on Camelot. It runs exceptionally well on tables. 
I want to be able to use 'guess' flavor that is marked as an enhancement in the issue#19
[]https://github.com/camelot-dev/camelot/issues/19[]

Do you have any update on that?
If I want to fix it, can you tell me what would be a good starting point in the code and otherwise?

**Comment by FrancoisHuet** [comment]
I would defer to Vinayak for proper guidance, but if you're interested I've taken a crack at a "hybrid" parser that attempts to combine the lattice approach with a text alignment one (similar to stream).
I've just recently tried to merge [my branch](https://github.com/FrancoisHuet/camelot/tree/hybrid-parser) with Vinayak's latest changes, so it's very much work in progress, but it could be a starting point for you if you want to do something similar -or if you'd like to give me feedback-.  Thanks!
François

**Comment by amansani** [comment]
Hello @vinayak-mehta , I am also looking for combination of stream and lattice flavor which is 'guess' falvor. 

In my case, i have the table in a pdf file, which is working properly with lattice flavor, but when i use stream falvor, the first two columns data are merging into one cell. 

In another case, i have a table in a pdf file, which is again giving all the results of a table using lattice flavor and missing last three rows of data when using stream flavor.

I cannot completely use lattice flavor, most of my tables don't have good borders and missing horizontal and vertical lines and the arrangement of data is better with stream flavor in output files when there is no borders and missing lines. So, now i am looking for some flavor which combines both of my cases.

In short,  i want to use lattice flavor where i have good table borders and  use stream flavor for rest of the tables. Is it possible to detect automatically and use the flavor which suits better for the table?

**Comment by VisheshJain112** [comment]
@vinayak-mehta I want automatically detect the flavor, any update on that part?

**Comment by bosd** [comment]
Will be implemented in next major release
